tests/test_autonomy/test_cli/test_push_all.py

```diff
@@ -55,8 +56,9 @@ def test_run(
         """Test run."""
 
         # packages/<author>/<component>/<name>/<config>.yaml
-        path = f"{PACKAGES}/*/*/*/*.yaml"
-        packages = {file.parent for file in self.t.glob(path)}
+        available_packages = len(
+            PackageManagerV1.from_dir(ROOT_DIR / PACKAGES).dev_packages
+        )
         published_packages = []
```

Comment:
why not `PackageManagerV1.from_dir(self.packages_dir).dev_packages`? We checked the temporary directory before, now we check the original packages directory instead. I don't think it makes a difference.

Reply:
Honestly this won't make a difference because we're dealing with `packages.json` which is going to be same in both root dir and temp dir, but I'll keep this in mind for future